### PR TITLE
Feature/#17

### PR DESCRIPTION
**Pull request by joetidee**
This will add the ability to supply breakpoints as em units. I was not sure how to test this before creating the pull request, as I know you are in the throws of putting some unit tests together - please advise.

### CODE REVIEW

**Comment by ehellman** [comment]
I'm going to test this out this week, however I don't think we should add a dependency (`polished`) for such small functions, these are both of them:

```js
function stripUnit(value) {
  const unitlessValue = parseFloat(value)
  if (isNaN(unitlessValue)) return value
  return unitlessValue
}

function em(pxValue) {
  const unitlessValue = stripUnit(value)
  return `${unitlessValue / 16}em`
}
```

I like that you added `px` as the default, and `em` as opt-in!

**Comment by joetidee** [comment]
Could really do with this in ASAP.

**Comment by joetidee** [comment]
I have made the changes to my local branch. When i run `git pull upstream master` I get the message 

> Permission denied (publickey).
> fatal: Could not read from remote repository.
> 
> Please make sure you have the correct access rights
> and the repository exists.

Could you help me out? I set up my local repo following this guide:
https://akrabat.com/the-beginners-guide-to-contributing-to-a-github-project/

**Comment by joetidee** [comment]
Is ok now, I set the remote to the https url as opposed to using SSH.

**Comment by joetidee** [comment]
Ok, have resolved conflicts and pushed changes.

**Comment by joetidee** [comment]
Any chance I can get this merged in today?

**Comment by ehellman** [comment]
For sure, testing this after work today and then I'll merge and release 3.0.2 :)

**Comment by joetidee** [comment]
Great. It would be good to get a heads-up on how I can also test it myself - could you document/supply me with your procedure? Are you any closer to having some test coverage - even if it is just one test, the integration of a chosen test suite would enable contributors to write some to test their work.

File: .babelrc [hunk 1]
```diff
@@ -43,6 +45,7 @@
         "react"
     ],
     "plugins": [
+        "babel-plugin-polished",
```
Comment (ehellman) [inline .babelrc]: These three occurrences prevent the project from compiling since the plugin doesn't exist anymroe

**Comment by ehellman** [comment]
Had to remove the polished plugin from `.babelrc`, but tested and everything works fine! Please remove it and push, I'll look in a couple of times during the day on Friday, merge and release 3.0.2 as soon as this is in :)

**Comment by ehellman** [comment]
As for the other questions, yes. Next on the agenda is getting the test suite merged and writing some guidelines for contributing! :)

**Comment by joetidee** [comment]
I have removed the plugin from .babelrc now :)

**Pull request merged**